tests/qplatform/test_device.py

```diff
@@ -0,0 +1,36 @@
+# Tests for device_topology:
```

Comment:
[nit]: This comment is misplaced. Not sure if we want to have the convention of having header comments for tests as the files seem to imply what they are tests for, so maybe change the name or remove.

Reply:
I removed it; and I removed the comments describing single tests, they are redundant given the test name is already explanatory